### PR TITLE
Add Sentry on the client side

### PR DESCRIPTION
**Pull request by rafenden**
## Description of change

Add sentry logger to our client-side code.

### CODE REVIEW

File: src/middleware/react-global-props.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const config = require('../config')
+
+module.exports = () => {
+  return function reactGlobalProps(req, res, next) {
+    res.locals.globalProps = {
+      sentryDns: config.sentryDsn,
```
Comment (web-bert) [inline src/middleware/react-global-props.js:6]: Typo here: `sentryDns` => `sentryDsn`

File: src/client/index.jsx [hunk 2]
```diff
@@ -211,8 +211,17 @@ function Mount({ selector, children }) {
   })
 }
 
+const appWrapper = document.getElementById('react-app')
+const globalProps = parseProps(appWrapper)
+
+if (globalProps.sentryDns) {
```
Comment (web-bert) [inline src/client/index.jsx:217]: Typo here: `sentryDns` => `sentryDsn`

File: src/client/index.jsx [hunk 3]
```diff
@@ -211,8 +211,17 @@ function Mount({ selector, children }) {
...
+  Sentry.init({
+    dsn: globalProps.sentryDns,
```
Comment (web-bert) [inline src/client/index.jsx:219]: Typo here: `sentryDns` => `sentryDsn`

File: src/middleware/react-global-props.js (hunk 1 above)
Comment (rafenden) [reply src/middleware/react-global-props.js]: Good spot, typo fixed!

File: src/client/index.jsx (hunk 2 above)
Comment (rafenden) [reply src/client/index.jsx]: Good spot, typo fixed!

File: src/client/index.jsx (hunk 3 above)
Comment (rafenden) [reply src/client/index.jsx]: Good spot, typo fixed!

**Pull request merged**